docs/source/distributed.rst

```diff
@@ -10,8 +10,9 @@ Distributed communication package - torch.distributed
 Backends
 --------
 
-``torch.distributed`` supports three backends, each with
-different capabilities. The table below shows which functions are available
+``torch.distributed`` supports three built-in backends, each with different
+capabilities.
+The table below shows which functions are available on GLOO/MPI/NCCL backends
```

Comment:
Please revert this, keeping only the "built-in" part. The explicit mention of backends is not needed.

Reply:
done